content/sensu-go/5.19/reference/backend.md

```diff
@@ -1273,7 +1285,7 @@ Here are some log rotate sample configurations:
   notifempty
   compress
   postrotate
-    kill -HUP `cat /var/run/sensu/sensu-backend.pid 2> /dev/null` 2> /dev/null || true
+    /sbin/service sensu-backend reload > /dev/null 2>/dev/null || true
```

Comment:
I think we need to revert the change to this line -- I've had a look at the sysvinit scripts and they don't implement the logic needed for `service sensu-backend reload` to work. The `kill -HUP` approach is correct for sysvinit.